examples/plotting/plot_2_parallel_coordinates.py

```diff
@@ -0,0 +1,148 @@
+"""
+====================
+Parallel Coordinates
+====================
+
+.. hint:: 
+
+   Conveys a dense overview of the trial objectives in a multi-dimensional space.
+   Helps identifying trends of best or worst hyperparameter values.
+
+The parallel coordinates plot decomposes a search space of `n` dimensions into `n`
+axis so that the entire space can be visualized simultaneously. Each dimension
+is represented as a vertical axis and trials are represented as lines crossing each 
+axis at the corresponding value of the hyperparameters. There is no obvious optimal ordering
+for the vertical axis, and you will often find that changing the order helps better understanding
+the data. Additionaly, the lines are plotted with graded colors based on the objective. The
+gradation is shown in a color bar on the right of the plot. Note that the objectives are added
+as the last axis is the plot as well.
+
+.. autofunction:: orion.plotting.base.parallel_coordinates
```

Comment:
There is a broken link to the plotly API docs in the generated document. Need to fix in source code for referenced function.